compiler/src/iree/compiler/Codegen/Common/test/math_transform_devicelib.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-codegen-math-transform{has-fast-exp})" --split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: func.func @erf_device_lib
+func.func @erf_device_lib(%arg0: f32) -> f32 attributes {
+  hal.executable.target = #hal.executable.target<"llvm-cpu", "xyz", {target_triple = "x86_64-xyz-xyz"}>
+} {
+  // CHECK-NOT: math.erf
+  // CHECK: scf.if
+  // CHECK: math.fma
+  // CHECK: scf.yield
+  // CHECK: else
+  // CHECK: math.fma
+  // CHECK: scf.yield
+  // CHECK: math.copysign
```

Comment:
This lit test should be expanded to verify
1. That the expected number of fmas are generated
2. That the expected coefficients are used